html/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Walter Velasquez</title>
+    <link rel="stylesheet" href="../css/style.css" />
+    <script src="https://kit.fontawesome.com/62ea397d3a.js" crossorigin="anonymous"></script>
+  </head>
+ 
+  <body>
+    <header>
+      <nav class="nav">
+        <div class="nav_mobile">
+          <div class="logo">Sir V</div>
+          <div class="nav_mobile-btn 
+          fas fa-bars"></div>
+        </div>
+      </nav>
+    </header>
+
+    <div class="headline">
+      <div class="presentation">
+        <div class="my_name">Walter Velasquez</div>
```

Comment:
It would be better to use `h1` tag for this main title instead of `div` tag to improve our structure and semantics.

Reply:
- [x] Issue well resolved ✔️